### ISSUE TITLE
(re)moving the Accessing episode?

### ISSUE THREAD

**Issue by bvreede** [carpentries-incubator/lesson-R-packaging#116]
The episode on Accessing Packages navigates in my opinion between teaching well known concepts (e.g. installing packages from CRAN and using `library()` to attach a package to the workspace) and addressing related but out of scope things like how to install a package from GitHub.

There is one crucial part, which is that functions should be called with their full name `packagename::functionname()` when used inside another package.

At the moment, this key information is buried.

I propose:
1. Moving the information on function names to the Functions episode
2. Moving the entire chapter to the back of the lesson, and adding some advice on how to distribute a package (i.e. hosting on GitHub; this can be done with drag and drop as well, so Git remains out of scope) 
3. OR, remove the chapter altogether.

@liekelotte @PabRod @n400peanuts happy to hear your thoughts!

**Comment by PabRod** [comment]
My thoughts, per point:

1. I agree.
2. and 3. I have mixed feelings. Starting with something familiar to the reader/student is not a bad idea. On the other hand, I am a big fan of simplifying as much as possible.

Hope it helps!

**Comment by multimeric** [comment]
I agree with this. The next time I run this course I intend to cut this section because I would rather have spent more time on vignettes than rehash these basics.

I have one other suggestion though, which is simply to make this episode optional. I don't mind actually keeping it as the second episode, because it's so fundamental. But it would make it more obvious to instructors that it's an easy skip.

**Comment by PabRod** [comment]
All options sound like reasonable ideas. I fully trust your choice, @multimeric.
